joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller.h

```diff
@@ -246,6 +252,54 @@ class JointTrajectoryController : public controller_interface::Controller<Hardwa
    */
   void setHoldPosition(const ros::Time& time, RealtimeGoalHandlePtr gh=RealtimeGoalHandlePtr());
 
+protected:
+  /**
+   * @returns the number of joints of the robot.
```

Comment:
For the Doxygen Html output it would be better to state what the function does via `@brief` statement instead of using the `@returns` statement. If `@brief` is used, Doxygen displays the explanation what the function does already in the summary at the beginning of the Html page. What do you think?

Reply:
I like `@brief` better. Done.